packages/jest-cli/src/get_node_handles.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {ProjectConfig} from 'types/Config';
+
+import {formatExecError} from 'jest-message-util';
+
+// Inspired by https://github.com/mafintosh/why-is-node-running/blob/master/index.js
+// Extracted as we want to format the result ourselves
+export default function collectHandles(): () => Array<Error> {
+  const activeHandles: Map<string, Error> = new Map();
+
+  function initHook(asyncId, type) {
+    const error = new Error(type);
```

Comment:
I'm using an error here for the free sourcemap support it provides. Then I don't need to talk to `runtime` to get them